src/extension/background-script/actions/allowances/add.ts

```diff
@@ -12,7 +14,7 @@ const add = async (message, sender) => {
     .first();
 
   if (allowance) {
-    await db.allowances.update(allowance.id, {
+    await db.allowances.update(allowance.id as number, {
```

Comment:
It could be `undefined`, yes? Could we just return if there's no `id`? Would be great if we can avoid `as` because it's not good practice.

Reply:
Done!